backend/orcid.py

```diff
@@ -293,7 +293,8 @@ def bulk_import(self, directory, fetch_papers=True, use_doi=False,
                         if fetch_papers:
                             papers = self.fetch_orcid_records(
                                 orcid,
-                                profile=OrcidProfile(json=profile),
+                                profile=OrcidProfile(orcid_id=orcid,
```

Comment:
Is it a valid use case to actually create an `OrcidProfile` without any `orcid_id`? If this is not the case, I would actually update the `OrcidProfile` constructor to avoid this kind of bugs to appear again.

Reply:
I think the intention that the constructor would indeed retrieve the ORCID id from the json representation if provided.